pages/linux/module.md

```diff
@@ -0,0 +1,28 @@
+# module
+
+> Modify a users' environment using the module command.
+> More information: <https://modules.readthedocs.io/en/latest/module.html>.
+
+- Display available modules:
+
+`module avail`
+
+- Search for a module by name:
+
+`module spider {{module_name}}`
```

Comment:
I can't find the `spider` subcommand in the more info link that you provided. Are you talking about `Lmod` ? <https://lmod.readthedocs.io/en/latest/>

Reply:
Yeah, that would be it. [https://lmod.readthedocs.io/en/latest/010_user.html](https://lmod.readthedocs.io/en/latest/010_user.html) might be a better link specific to the lmod module command in terms of function rather than the docs for the full lmod ecosystem?. Would it be better practice in the TLDR repository to include the page more specific to the ecosystem or to `module` itself?

Reply:
Is the whole page about lmod, or just this example? I'm somewhat confused here (despite the fact that I use the `module` on my University's HPC).

Reply:
The page @navarroaxel posted is for all of Lmod itself. The link in my previous comment is on the module portion itself user side rather than the background that app specialists would handle like the module files. For the sake of `module` in general use, the link I sent, I believe, would be better, but either would be appropriate.

Reply:
Ah, I see. I'm still somewhat confused, but let's go with the one you think would be more appropriate here.

Reply:
Sounds good.